javaslang/src/main/java/javaslang/collection/List.java

```diff
@@ -941,7 +941,7 @@ default T peek() {
         final Deque<T> preceding = new ArrayDeque<>(size());
         List<T> result = this;
         boolean found = false;
-        while (!found && !result.isEmpty()) {
+        while (!found && result.isNotEmpty()) {
```

Comment:
@paplorinc sorry for having ruined this PR! You are very fast with refactorings in the broad - it is hard for me to follow and spend enough time to think about changes. Keep up the awesome work! You already have made significant changes to Javaslang!